### ISSUE TITLE
Internal: Clean up `use`s

### ISSUE THREAD

**Issue by KyGost** [gluesql/gluesql#174]
It would be nice to go through and clean up usages of `use` so that they are all like:
```rust
use {
    super::{
        create_table::validate_table,
        fetch::{fetch, fetch_columns},
        filter::Filter,
        select::{select, select_with_labels},
        update::Update,
        validate::{validate_rows, ColumnValidation},
    },
    crate::{
        data::{get_name, Row, Schema},
        parse_sql::Query,
        result::{MutResult, Result},
        store::{AlterTable, AutoIncrement, Store, StoreMut},
    },
    sqlparser::ast::{ObjectType, SetExpr, Statement, Values},
};
```
instead of
```rust
use sqlparser::ast::{ObjectType, SetExpr, Statement, Values};

use crate::data::{get_name, Row, Schema};
use crate::parse_sql::Query;
use crate::result::{MutResult, Result};
use crate::store::{AlterTable, AutoIncrement, Store, StoreMut};

use super::create_table::validate_table;
use super::fetch::{fetch, fetch_columns};
use super::filter::Filter;
use super::select::{select, select_with_labels};
use super::update::Update;
use super::validate::{validate_rows, ColumnValidation};
```

**Comment by panarch** [comment]
I agree, previous `use` has some implicit rules and it could be bothersome for random contributors.
Using `use { .. }`, `cargo fmt` simply takes the whole role and we don't need to care about arranging `use`.
Let `cargo fmt` does everything :) Thanks for the good suggestion.

**Comment by Butuhpakagsial** [comment]
https://github.com/gluesql/gluesql/issues/174#issue-830771998

**Comment by KyGost** [comment]
I've just been looking through [rustfmt](https://github.com/rust-lang/rustfmt)'s documentation and found a super awesome feature!
[imports_granularity](https://rust-lang.github.io/rustfmt/#imports_granularity) automatically does this. It is currently unstable and the `One` option (how I tend to prefer it) is even more so but once done it should remove this concious choice :-)

**Comment by panarch** [comment]
@KyGost Wow, thanks for sharing. Once it comes into stable, then it would be good to use this option.
I just tried `merge_imports` but it looks like.. both `imports_granularity` and `merge_imports` work only in Rust nightly.

btw, it was supposed to be closed by the pr #223, closing now manually.
